website/bitcoin_utils.py

```
@@ -58,3 +60,51 @@ def issue_asset(asset_name, amount, identifier):
     return txid
+
+
+def send_bch_payment(address, amount):
+    """
+    Send BCH payment using your BCH payment provider.
+    Returns transaction ID.
+    """
+
+    # Validate BCH address format
+    if not address.startswith("bitcoincash:"):
+        raise ValueError(f"Invalid BCH address: {address}")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)